generator/source/faq/settings.rst

```diff
@@ -40,15 +40,29 @@ Since Sawtooth settings are extensible and include transaction family-specific s
 sawtooth.config.authorization_type
     Example setting--never used.  To set authorization type, use command line option ``sawtooth-validator --network-auth {trust|challenge}``
 
-sawtooth.consensus.algorithm
-    Consensus algorithm (e.g., ``poet`` (PoET SGX or PoET CFT) or ``devmode`` (default) or ``raft`` or any other pluggable consensus engine you provide)
+sawtooth.consensus.algorithm.name
+    Pluggable consensus algorithm name. These include ``PoET``, ``Devmode``, ``sawtooth-raft-engine``, and ``sawtooth-pbft``.  The default is ``devmode`` for Sawtooth 1.1 or earlier, with no default for the (unreleased) nightly build.
+sawtooth.consensus.algorithm.version
+    Consensus algorithm version. Currently 0.1 for PoET, Devmode, and sawtooth-pbft, and 0.1.0 for sawtooth-raft-engine.
 sawtooth.consensus.block_validation_rules
     Lists validation rules to use in deciding what blocks to add to the blockchain.
     See https://sawtooth.hyperledger.org/docs/core/nightly/master/architecture/injecting_batches_block_validation_rules.html
 sawtooth.consensus.max_wait_time
     Maximum devmode consensus wait time, in seconds
 sawtooth.consensus.min_wait_time
     Minimum devmode consensus wait time, in seconds
+
+sawtooth.consensus.pbft.block_duration
```

Comment:
These settings are out of date. please see https://github.com/hyperledger/sawtooth-pbft/pull/85/files#diff-6d8a5e2104954a026de2bfe2b936aea4L78 for the updated ( and soon to be merged) most updated version.